plugins/source/awspricing/docs/tables/awspricing_service_products.md

```diff
@@ -0,0 +1,19 @@
+# Table: awspricing_service_products
+
+The primary key for this table is **sku**.
```

Comment:
Just a question: are there any good URLs we can link to for the table descriptions that describe the columns beyond what can be seen here?